src/scribe_data/wiktionary/parse_dump.py

```diff
@@ -101,120 +102,19 @@ def _build_iso_mapping(self) -> dict:
             if iso_code := data.get("iso"):
                 iso_mapping[iso_code] = lang_name
 
-        return iso_mapping
-
-    # MARK: process total
-    def _process_lexeme_total(self, lexeme: dict) -> None:
-        """
-        Gather stats if 'total' is in parse_type: how many entries per language & category,
-        how many translations, etc.
-        """
-        lexicalCategory = lexeme.get("lexicalCategory")
-        if not lexicalCategory or lexicalCategory not in data_type_metadata.values():
-            return
-
-        category_name = self._category_lookup.get(lexicalCategory)
-        if not category_name:
-            return
-
-        # Update counters.
-        lemmas = lexeme.get("lemmas", {})
-        for lemma in lemmas.values():
-            lang = lemma.get("language")
-
-            if lang in self.iso_to_name:
-                self.lexical_category_counts[lang][category_name] += 1
-                translation_count = sum(
-                    len(sense.get("glosses", {})) for sense in lexeme.get("senses", [])
-                )
-                self.translation_counts[lang][category_name] += translation_count
-
-                break
-
-    # MARK: process translations
-    def _process_lexeme_translations(self, lexeme: dict) -> None:
-        """
-        Process gloss-based translations if 'translations' is in parse_type.
-        Store them in self.translations_index.
-        """
-        lemmas = lexeme.get("lemmas", {})
-        qid = lexeme.get("lexicalCategory")
-
-        if not (lemmas and qid):
-            return
-
-        category_name = self._category_lookup.get(qid)
-        if not category_name:
-            return
-
-        # Only store first valid lemma for translations.
-        for lang_code, lemma_data in lemmas.items():
-            if lang_code not in self.iso_to_name:
-                continue
-
-            word = lemma_data.get("value", "").lower()
-            if not word:
-                continue
-
-            # Build translations from sense glosses.
-            translations = {}
-            for sense in lexeme.get("senses", []):
-                for sense_lang_code, gloss in sense.get("glosses", {}).items():
-                    if sense_lang_code in self.iso_to_name:
-                        translations[sense_lang_code] = gloss["value"]
+        for language in self.target_iso:
+            if (
+                language.startswith("Q")
+                or language.startswith("q")
```

Comment:
```suggestion
                language.lower().startswith("q")
```
Could do the above potentially? 

nit - just a suggestion really